SUMMARY.md

```diff
@@ -0,0 +1,40 @@
+## Table of Contents
+
+* [Ternary Operator](content/TernaryOperator/TernaryOperator.q.md) ** - minutes**
+    > The ternary operator is used as a shorthand way of doing `if/else` statements. In this lesson, you will learn how to use the ternary operator to perform shorthand comparisons in PHP.
+
+* [Type Hinting](content/TypeHinting/TypeHinting.q.md) ** - minutes**
+    > Discover how you can use type hinting to define the data type that is passed in a function declaration.
+
+* [Cookies](content/Cookies/Cookies.q.md) ** - minutes**
+    > Learn how to create, retrieve, modify, and delete cookies using the `setCookie()` function.
+ 
+* [Sessions](content/Sessions/Sessions.q.md) ** - minutes**
+    > Learn how to use session functions to start, get, modify, and destroy a session.
+
+* [Serialization](content/Serialization/Serialization.q.md) ** - minutes**
+    > Serialization in PHP is used for storing and passing values while retaining their type and structure. In this lesson, you will learn how to use `serialize()` and `unserialize()` functions in PHP.
+
+* [Error Handling](content/ErrorHandling/ErrorHandling.q.md) ** - minutes**
+    > Error handling refers to the process of catching errors detected in the program that requires proper action. Find out how you can handle errors using different types of error handling functions in PHP.
+    
+* [Catching Exceptions](content/CatchingExceptions/CatchingExceptions.q.md) ** - minutes**
+    > In this lesson, you will learn how to use the basic PHP `Exception` classes and the `try` and `catch` statements in handling errors.
+ 
+* [Throwing Exceptions](content/ThrowingExceptions/ThrowingExceptions.q.md) ** - minutes**
+    > Discover how you can throw a new exception using the `throw` statement. 
```

Comment:
You can throw existing exception, it's called rethrow, so throwing exception doesn't have to a `new` exception.